### PR TITLE
tony/mpl hybrid docs rewrite

### PR DESCRIPTION
**Pull request by tonyboylehub**
- New SDK section.
- Stripped back version of the docs into their own key sections.
- Should be merged along side new guide which takes the place of the more detailed content that was adjusted.

### CODE REVIEW

File: src/pages/mpl-hybrid/create-escrow.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+{% /totem-accordion %}
+{% /totem %}
+
+## Create Escrow
+
+### Args
+
+#### name
+
+The name of your escrow.
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/create-escrow.md:51]: what is it important for? Maybe provide more context

File: src/pages/mpl-hybrid/create-escrow.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+```ts
+name: 'My Test Escrow'
+```
+
+#### uri
+
+This is the base uri for your metadata pool. This needs to be a static uri which also contains your metadata json files at sequential destination. ie:
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/create-escrow.md:59]:
```suggestion
This is the base uri for your metadata pool. This needs to be a static uri which also contains your metadata json files at sequential destination. i.e.:
```

File: src/pages/mpl-hybrid/create-escrow.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+path: rerollEnabled,
+```
+
+#### associatedTokenProgram
+
+The `SPL_ASSOCIATED_TOKEN_PROGRAM_ID` can be pulled from the `mpl-toolbox` package.
+
+```ts
+// Associated Token Program ID
+associatedTokenProgram: SPL_ASSOCIATED_TOKEN_PROGRAM_ID,
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/create-escrow.md:181]: maybe add the import?

File: src/pages/mpl-hybrid/funding-escrow.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: Funding the MPL Hybrid 404 Escrow
+metaTitle: Funding the MPL Hybrid 404 Escrow | MPL-Hybrid
+description: Learn to fund the MPL 404 Hybrid Escrow account with SPL Tokens that makes 404 swaps possible.
+---
+
+The next step before the smart-swap is live it to fund the escrow. Typically if a project wants to ensure the escrow always stays funded, they start by releasing all of the NFTs or tokens and then placing all of the other assets in the escrow. This ensures that every outstanding asset is "backed" by the counter-asset in the escrow. Because the Escrow is a PDA, loading it via wallets is not widely supported. You can use the below code to transfer assets into your escrow.
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/funding-escrow.md:7]: maybe language barrier, but "live" sounds wrong?

File: src/pages/mpl-hybrid/update-escrow.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+const escrowConfigurationAddress = publicKey("11111111111111111111111111111111");
+
+// Fetch the escrow configuration account.
+const escrowConfigurationData = await fetchEscrowV1(umi, escrowConfigurationAddress);
+
+// Use the spread operator `...` to spread the `escrowConfigurationData` fields into the object
+// and adjust any fields you wish to update.
+const res = await updateEscrowV1(umi, {
+    ...escrowConfigurationData,
+    escrow: hybridEscrowAddress,
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/update-escrow.md:23]: should we mention wherei t is from? Similar to the escrowConfigurationAddress?

File: src/pages/mpl-hybrid/funding-escrow.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+// Generate the Token Account PDA from the funding wallet.
+const sourceTokenAccountPda = findAssociatedTokenPda(umi, {
+  owner: umi.identity.publicKey,
+  mint: tokenMint,
+})
+
+// Generate the Token Account PDA for the escrow destination.
+const escrowTokenAccountPda = findAssociatedTokenPda(umi, {
+  owner: hybridEscrowAddress,
```
Comment (MarkSackerberg) [inline src/pages/mpl-hybrid/funding-escrow.md:25]: should we mention wherei t is from? Similar to the escrowAddress?

**Review by MarkSackerberg — CHANGES_REQUESTED** [review]
grafik
Something is wrong in the hero

File: src/pages/mpl-hybrid/sdk/javascript.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+You can add the `mplHybrid()` package anywhere in your umi instance creation.
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+import { mplHybrid } from '@metaplex-foundation/mpl-hybrid'
+
+// Use the RPC endpoint of your choice.
+const umi = createUmi('http://api.devenet.solana.com')
+... // additional umi settings and packages
+.use(mplHybrid())
```
[the remark this answers is not in the document]
Comment (tonyboylehub) [reply src/pages/mpl-hybrid/sdk/javascript.md:39]: Hmm. I think that's probably for token creation. I'm not doing that here in these intial docs.

File: src/pages/mpl-hybrid/create-escrow.md (hunk 1 above)
Comment (tonyboylehub) [reply src/pages/mpl-hybrid/create-escrow.md:51]: I think it has zero importance/impact to be honest other than its just a title that you can perhaps pull and display on a UI if you want to. It's not part of the seed for the escrow, not part of the naming process of released nfts (that gets changed to just "Captured".

Will add the UI name part in.

File: src/pages/mpl-hybrid/update-escrow.md (hunk 5 above)
Comment (tonyboylehub) [reply src/pages/mpl-hybrid/update-escrow.md:23]: good spot. This was me renaming and forgetting to rename some other stuff.

File: src/pages/mpl-hybrid/funding-escrow.md (hunk 4 above)
Comment (tonyboylehub) [reply src/pages/mpl-hybrid/funding-escrow.md:7]: It's wrong anyway

Should be "The next step before the smart-swap is live **is** to fund the escrow".

Will adjust this sentance.

**Comment by tonyboylehub** [comment]
Banner image fixed

**Review by MarkSackerberg — APPROVED** [review]

**Review by danenbm — APPROVED** [review]

**Pull request merged**